RecoCTPPS/Configuration/test/raw_data_test.py

```diff
@@ -4,39 +4,45 @@
 
 # minimum of logs
 process.MessageLogger = cms.Service("MessageLogger",
-    statistics = cms.untracked.vstring(),
-    destinations = cms.untracked.vstring('cout'),
-    cout = cms.untracked.PSet(
-        threshold = cms.untracked.string('WARNING')
-    )
+  statistics = cms.untracked.vstring(),
+  destinations = cms.untracked.vstring('cout'),
+  cout = cms.untracked.PSet(
+    threshold = cms.untracked.string('WARNING')
+  )
 )
 
 # raw data source
 process.source = cms.Source("PoolSource",
-    fileNames = cms.untracked.vstring('file:/afs/cern.ch/user/j/jkaspar/public/run273062_ls0001-2_stream.root')
+  fileNames = cms.untracked.vstring(
+    'file:/afs/cern.ch/user/j/jkaspar/public/run273062_ls0001-2_stream.root',
```

Comment:
Is the lifetime of the diamond detectors so short that they won't outlive this personal file under your home directory? Shouldn't we have a more standard file here?